gitbase-playground/templates/statefulset.yaml

```diff
@@ -0,0 +1,157 @@
+        accessModes: [ "ReadWriteOnce" ]
+        resources:
+          requests:
+            storage: "{{ .Values.gitbaseServer.storage.size }}"
+    - metadata:
+        name: gitbase-repos-volume
+      spec:
+      {{- if .Values.gitbaseServer.storage.class }}
+      {{- if (eq "-" .Values.gitbaseServer.storage.class) }}
+        storageClassName: ""
+      {{- else }}
+        storageClassName: "{{ .Values.gitbaseServer.storage.class }}"
+      {{- end }}
+      {{- end }}
+        accessModes: [ "ReadWriteOnce" ]
+        resources:
+          requests:
+            storage: "{{ .Values.gitbaseServer.storage.size }}"
+  template:
+    metadata:
+      labels:
+        app: {{ template "gitbase-playground.name" . }}
+        release: {{ .Release.Name }}
+      # This is to upgrade the deployment if configmap changes
+      annotations:
+        checksum/install-bblfshd-drivers: {{ include (print $.Template.BasePath "/install-bblfshd-drivers-configmap.yaml") . | sha256sum }} 
+        checksum/clone-git-repositories: {{ include (print $.Template.BasePath "/clone-git-repositories-configmap.yaml") . | sha256sum }} 
+    spec:
+      volumes:
+        - configMap:
+            name: {{ template "gitbase-playground.fullname" . }}-install-bblfshd-drivers
+            items:
+            - key: install-bblfshd-drivers.sh
+              path: install-bblfshd-drivers.sh
+          name: install-bblfshd-drivers-volume
+        - configMap:
+            name: {{ template "gitbase-playground.fullname" . }}-clone-git-repositories
+            items:
+            - key: clone-git-repositories.sh
+              path: clone-git-repositories.sh
+          name: clone-git-repositories-volume
+      initContainers:
+        - name: clone-git-repositories
+          image: "{{ .Values.gitbaseServer.image.repository }}:{{ .Values.gitbaseServer.image.tag }}"
+          imagePullPolicy: IfNotPresent
+          command: [ 'sh', '/opt/clone-git-repositories.sh' ]
+          volumeMounts:
+            - name: clone-git-repositories-volume
+              mountPath: /opt/clone-git-repositories.sh
+              subPath: clone-git-repositories.sh
+            - name: gitbase-repos-volume
+              mountPath: {{ .Values.gitbaseServer.repositoriesDir }}
+      containers:
+        - name: gitbase-playground
+          image: "{{ .Values.gitbasePlayground.image.repository }}:{{ required "Missing gitbasePlayground.image.tag" .Values.gitbasePlayground.image.tag }}"
+          imagePullPolicy: {{ .Values.gitbasePlayground.image.pullPolicy }}
+          env:
+            - name: PORT
+              value: "{{ .Values.service.internalPort }}"
+            - name: GITBASEPG_ENV
+              value: "{{ .Values.gitbasePlayground.env }}"
+            - name: GITBASEPG_DB_CONNECTION
+              value: "gitbase@tcp(127.0.0.1:3306)/none?maxAllowedPacket={{ .Values.gitbasePlayground.gitbaseMaxAllowedPacket }}"
+            - name: GITBASEPG_BBLFSHD_URL
```

Comment:
Same as before, it should be [`GITBASEPG_BBLFSH_SERVER_URL`](https://github.com/src-d/gitbase-playground/blob/master/docs/CONTRIBUTING.md#configuration)

Reply:
fixed in 8a1038b